src/fish_ai/config.py

```diff
@@ -20,6 +22,9 @@ def get_config(key):
     active_section = config.get(section='fish-ai', option='configuration')
 
     if config.has_option(section=active_section, option=key):
+        # special case for 'api-key'
+        if use_keyring and key == 'api_key':
```

Comment:
The Huggingface provider uses `password` instead of `api_key`. We should handle that case as well.